projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -15,30 +15,33 @@ import InboxList from './list';
 import InboxResponse from './response';
 import './style.scss';
 
-const RESPONSES_FETCH_LIMIT = 20;
+const RESPONSES_FETCH_LIMIT = 5;
 
 const Inbox = () => {
 	const [ currentResponseId, setCurrentResponseId ] = useState( -1 );
-	const [ searchText, setSearchText ] = useState( '' );
-	const [ currentPage, setCurrentPage ] = useState( 1 );
-	const [ searchTerm, setSearchTerm ] = useState( searchText );
 	const [ view, setView ] = useState( 'list' );
 
-	const { invalidateResolution } = useDispatch( STORE_NAME );
+	const { invalidateResolution, setSearch } = useDispatch( STORE_NAME );
+
+	const search = useSelect( select => select( STORE_NAME ).getSearch() );
+
+	const [ searchText, setSearchText ] = useState( search );
+	const [ currentPage, setCurrentPage ] = useState( 1 );
```

Comment:
I think eventually we'd want to move this into the state as well and keep it with the rest of the query. Certainly not something for this PR but I thought I'd note it.

Reply:
I originally did, but then I got into this whole mess of effect dependencies, where the effect would trigger twice due to search and page changing at the same time. As a continuation of our slack chat: is the page something we need on the state?

Agreed, for another PR.